_docs-sources/iac/whats-this/index.md

```diff
@@ -1,5 +1,27 @@
 # What is the Infrastructure as Code Library?
 
-Qu'est-ce que c'est?
+The Gruntwork Infrastructure as Code Library (IaC Library) is a collection of reusable code that enables you to deploy and manage infrastructure quickly and reliably. It promotes code reusability, modularity, and consistency in infrastructure deployments. We've taken the thousands of hours we spent building infrastructure on AWS and condensed all that experience and code into pre-built packages or modules.  
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Morbi leo urna molestie at. Volutpat blandit aliquam etiam erat. In massa tempor nec feugiat nisl pretium. Vitae auctor eu augue ut lectus arcu bibendum at. Auctor augue mauris augue neque gravida. Vulputate sapien nec sagittis aliquam malesuada bibendum arcu. Bibendum est ultricies integer quis auctor elit. Morbi tincidunt augue interdum velit euismod in pellentesque. Imperdiet proin fermentum leo vel orci porta non pulvinar neque. Felis bibendum ut tristique et egestas quis ipsum. In arcu cursus euismod quis viverra nibh cras pulvinar mattis. Massa massa ultricies mi quis hendrerit dolor. Vel pharetra vel turpis nunc eget lorem. Tortor consequat id porta nibh venenatis cras. Nunc pulvinar sapien et ligula ullamcorper. Vehicula ipsum a arcu cursus vitae congue mauris rhoncus. Sit amet porttitor eget dolor morbi non.
+The library consists of two types of code: Modules & Services
+
+## Modules
+
+Modules are reusable code components that are used to deploy and manage specific pieces of infrastructure. These modules encapsulate the configuration and resource definitions required to create and manage a particular component, such as a VPC, ECS cluster, or an Auto Scaling Group. For more information on modules check out the [Modules page](/iac/whats-this/modules/).
+
+## Services
+
+Services in the service catalog are reusable code that combines multiple modules from the IaC Library. It simplifies the deployment and management of complex infrastructure configurations and rather than dealing with individual modules and their dependencies, users can directly deploy services tailored for a particular use case. 
```

Comment:
```suggestion
Services in the service catalog are reusable code that combines multiple modules from the IaC Library, simplifying the deployment and management of complex infrastructure configurations. Rather than dealing with individual modules and their dependencies, users can directly deploy services tailored for a particular use case. 
```